velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -120,8 +120,12 @@ ::duckdb::Value duckValueAt<TypeKind::BIGINT>(
   }
 
   if (type->isIntervalDayTime()) {
-    return ::duckdb::Value::INTERVAL(
-        0, 0, vector->as<SimpleVector<int64_t>>()->valueAt(index));
+    static constexpr int64_t kMicrosecondsInDay =
+        1000L * 1000L * 60L * 60L * 24L;
+    auto interval = vector->as<SimpleVector<int64_t>>()->valueAt(index);
```

Comment:
```
nit: auto* interval
```

Reply:
valueAt from a SimpleVector<int64_t> returns an int64_t not a pointer